docs/data/product/dea-surface-reflectance-landsat-7-etm/_tables.yaml

```
@@ -74,7 +74,7 @@ bands_table:
     nodata: -999
     units: null
     type: int16
-    description: null
+    description: "The panchromatic band is high resolution and can be used for pan-sharpening. Wavelength: 0.50–0.68 µm."
```

@robbibt Is this correct? (I have already checked all the wavelengths are correct, i'm only asking for a review of the text.)

---

```suggestion
    description: "The panchromatic band is high resolution and can be used for pan-sharpening Landsat 7's green, red and NIR bands. Wavelength: 0.52–0.90 µm."
```

The wavelength here is actually not quite right - Landsat 7 band wavelengths (particularly for panchromatic) are different from Landsat 8 and 9, and overlap different bands. The equivalent wording for Landsat 8 and 9 would be:
```
    description: "The panchromatic band is high resolution and can be used for pan-sharpening Landsat 8 and 9's blue, green and red bands. Wavelength: 0.50–0.68 µm."
```

---

Wow, I didn't notice that. Ok that's good, I've now applied your changes.